samples/graph-change-notification-team-channel/csharp/ChangeNotification/Provider/GraphBetaClient.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace ChangeNotification.Provider
+{
+    using Azure.Identity;
+    using ChangeNotification.Model.Configuration;
+    using Microsoft.Extensions.Options;
+    using Microsoft.Graph;
+
+    public class GraphBetaClient
```

Comment:
Why do we need Beta client ?
Check the Microsoft Doc whether we the required API are in V1.0 or in Beta ?